### ISSUE TITLE
Automerge

### ISSUE THREAD

**Issue by mattlenz** [ipfs/research-CRDT#24]
Automerge (https://github.com/automerge/automerge) uses CRDTs. There is a discussion on Hacker News (https://news.ycombinator.com/item?id=16309533)

**Comment by mattlenz** [comment]
and a related paper https://arxiv.org/abs/1608.03960

**Comment by diasdavid** [comment]
@pgte have you looked into this one? Also https://github.com/automerge/automerge/issues/54

**Comment by pgte** [comment]
Yes I have, throughly. It's a very interesting CRDT that minimizes data loss over deeply nested JSON structures. Looking at this specific issue now..
